### PR TITLE
Refactor movie

### PR DESCRIPTION
**Pull request by lourdesVillca**

### CODE REVIEW

**Comment by lourdesVillca** [comment]
@carledriss ready for code review

File: src/main/java/org/fundacionjala/movies/Main.java [hunk 1]
```diff
@@ -0,0 +1,11 @@
+package org.fundacionjala.movies;
+
+public class Main {
```
Comment (carledriss) [inline src/main/java/org/fundacionjala/movies/Main.java]: Remove this unnecessary class

**Comment by carledriss** [comment]
@lourdesVillca 
Please apply the changes that we did in the class

**Comment by lourdesVillca** [comment]
@carledriss ready to code review

**Comment by carledriss** [comment]
@lourdesVillca 
You can still refactor the code.
Take a look again.
:)

**Comment by lourdesVillca** [comment]
@carledriss ready to code review

File: src/main/java/org/fundacionjala/movies/ManageRental.java [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+        return rentals.stream()
+                .mapToDouble(Rental::calculateAmount).sum();
+    }
+
+    /**
+     * Method that builds the result.
+     *
+     * @return the statement with the result.
+     */
+    public String statement() {
```
Comment (carledriss) [inline src/main/java/org/fundacionjala/movies/ManageRental.java:61]: It is missing the unit test for this method. :(

**Comment by lourdesVillca** [comment]
@carledriss ready to code review

**Comment by carledriss** [comment]
Looks good to me!

**Pull request merged**